lib/Elevate/Components.pm

```diff
@@ -75,11 +98,38 @@ our @BLOCKERS = qw{
   AutoSSL
 };
 
-push @BLOCKERS, 'Leapp';    # This blocker has to run last!
+push @CHECKS, 'Leapp';    # This blocker has to run last!
```

Comment:
The comment is no longer true. Does it still need to be?

Reply:
It does.  I have correct this.